### PR TITLE
fix: update fields

### PR DESCRIPTION
**Pull request by izabellalarsson**
https://trello.com/c/YBx7yg08/256-remove-minimum-time-restrictions-when-adding-a-new-transport

### CODE REVIEW

**Review by momentiris — COMMENTED** [review]
Does this mean booking time restriction also is not required?

**Comment by izabellalarsson** [comment]
The time restriction for a booking isn't required. I see that we might want it to be required if you checked the checkbox.

File: packages/engine-ui/src/components/forms/inputs/DateInput.tsx [hunk 1]
```diff
@@ -21,10 +21,19 @@ const TimeRestrictionDateInput = React.forwardRef<
...
   (
-    { onChange, onClick, value, placeholder, withIcon = true, handleFocus },
+    {
+      onChange,
+      onClick,
+      value,
+      placeholder,
+      withIcon = true,
+      handleFocus,
+      shouldBeRequired = true,
```
Comment (lundeeell) [inline packages/engine-ui/src/components/forms/inputs/DateInput.tsx:35]: Do we want default to be required? Think opposite is more common

Comment (lundeeell) [reply packages/engine-ui/src/components/forms/inputs/DateInput.tsx:35]: Also, naming, `should` is kind of vague, think just `required` would do it, a bit more straightforward?

**Review by lundeeell — APPROVED** [review]
Works nice! However, see comments before merging!

File: packages/engine-ui/src/components/forms/inputs/DateInput.tsx (hunk 1 above)
Comment (momentiris) [reply packages/engine-ui/src/components/forms/inputs/DateInput.tsx:35]: Doesn't work to name it required, gets lost some way in the prop/forwardref.. 🤷‍♂️

Comment (lundeeell) [reply packages/engine-ui/src/components/forms/inputs/DateInput.tsx:35]: Then, isRequired?

**Pull request merged**